internal/manifests/mutate.go

```diff
@@ -56,6 +58,7 @@ var (
 // - HorizontalPodAutoscaler
 // - Route
 // - Secret
+// - TargetAllocator
```

Comment:
to confirm my understanding here... we need this because the collector CR will directly create the TA CR and that will be part of the mutate loop now?

Reply:
Yes, exactly.